src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js

```diff
@@ -618,5 +620,14 @@
         }
     });
 
+    $('.ez-tabs a').on('shown.bs.tab', (event) => {
```

Comment:
We avoid using elements selectors and tend to use CSS classes instead:
```suggestion
    $('.ez-tabs .nav-link').on('shown.bs.tab', (event) => {
```